lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -638,7 +637,7 @@ def setUpClass(cls):
                 metadata={'graded': True, 'format': 'Homework'})
             for _ in xrange(4)
         ]
-        problems = [
+        cls.problems = [
```

Comment:
If there's non-obvious value in having the attribute, then keep it, and add a comment.